burr/cli/__main__.py

```diff
@@ -358,17 +383,60 @@ def create_test_case(
     from burr.tracking.client import LocalTrackingClient
 
     local_tracker = LocalTrackingClient(project=project_name)
-    data: PersistedStateData = local_tracker.load(
-        partition_key=partition_key, app_id=app_id, sequence_id=int(sequence_id)
+    sequence_id = int(sequence_id)
+
+    after_action: PersistedStateData = local_tracker.load(
+        partition_key=partition_key, app_id=app_id, sequence_id=sequence_id
     )
-    if not data:
-        print(f"No data found for {app_id} in {project_name} with sequence {sequence_id}")
+    if not after_action:
+        print(f"No state found for {project_name}, {partition_key}, {app_id}, {sequence_id}.")
         return
-    state_dict = data["state"].serialize()
-    print("Found data for action: ", data["position"])
+
+    action_name = action_name if action_name else after_action["position"]
+    """
+    Explanation of the logic here.
+    If there's an error:
+     - status would be failed vs completed
+     - state loaded would be the starting state
+    If there's no error:
+     - status would be completed
+     - state loaded would be the ending state, so to get the starting state, we need to look at the prior sequence ID
+    """
+    if after_action["status"] == "completed":
+        print("Action was successful so loading initial and expected state into test fixture.")
```

Comment:
We should have logger.warn instead of print here as we use stdout.

Reply:
prints are used throughout here.

Reply:
We shouldn't -- it sometimes will/sometimes won't parse, and you'll be putting that to a file. You want to be able to pipe it.